src/ZkMinterERC1155V1.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {IERC1155} from "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
```

Comment:
@alexkeating Should we use this or just create a new interface. 
I ask this because the only function we need from this contract is `balanceOf(address _user, uint256 _tokenId)`